collections/Don/MS_Don_e_250.xml

```diff
@@ -71,10 +76,441 @@
+                        </supportDesc>
+                     </objectDesc>
+                  </physDesc>
+                 
+               </msPart>
+               <msPart xml:id="MS_Don_e_250-partA">
+                  <msIdentifier>
+                     <altIdentifier type="partial">
+                        <idno type="part">MS. Don. e. 250 - part A (fols. 1-34)</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem n="4" xml:id="MS_Don_e_250-partA-item1">
+                        <locus>(fols 1r–15v)</locus>
+                        <title type="desc" key="work_11337"> Gospel harmony of the Passion of Christ for the liturgical hours</title>
+                        <rubric>Concordancia Ewangelystarum. De passione domini</rubric>
+                        <incipit>Appropinquante passionis tempore et iam habito consilio inter principes sacerdotum cum Iuda</incipit>
+                        <explicit>Illi autem abeuntes munierunt sepulchrum signantes lapidem cum custodibus. Deo gracias etc</explicit></msItem>
+                     <msItem n="5" xml:id="MS_Don_e_250-partA-item2">
+                        <locus>(fols 16r–32v)</locus>
+                        <author key="person_84971788"><supplied>JORDAN OF QUEDLINBURG</supplied></author>
+                        <title key="work_2867"><supplied>Meditationes de passione Christi alias Articuli LXV de passione Domini cum theorematibus et documentis</supplied></title>
+                        <note>Excerpts, <seg type="addition" resp="#MLH">consisting of the 'orationes' only of the 65 sections, numbered in arabic in the margin, sometimes followed by an additional prayer for the same occasion.</seg></note><note>For the author see <hi rend="superscript">2</hi>VL, IV (1983), cols 853–861; for the transmission of the text see A. Zumkeller, <title>Manuskripte von Werken der Autoren des Augustiner-Eremitenordens in mitteleuropäischen Bibliotheken</title>, pp. 293–298).</note>
+                        <rubric>Ad primum completorium</rubric>
+                        <incipit>Domine ihesu criste fili dei viui qui hora completorij pro me misero peccatore imminente passione tua tristarj pauere et mestus esse voluisti</incipit>
+                        <explicit>et ego tibi consepultus peruenire tecum merear  ad resurrectionis gloriam. Amen</explicit></msItem>
+                     <msItem n="6" xml:id="MS_Don_e_250-partA-item3">
+                        <locus>(fols 32v–34v)</locus>
+                        <title type="desc" key="work_13950">Prayers</title>
+                        <msItem>
+                           <locus>(fol. 32v–33r)</locus>
+                           <rubric> Oracio ad wlnera cristi</rubric>
+                           <incipit>Exaltabo te domine qui suscepisti me  O Solutifera wlnera dulcissimi  amatoris mei ihesu cristi</incipit>
```

Comment:
Are these extra spaces meant to represent line breaks?

Reply:
They may represent places where I removed existing line breaks (where the description was recording each new line of the manuscript)